backend/presentation/views.py

```diff
@@ -0,0 +1,48 @@
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.viewsets import ModelViewSet
+
+from presentation.models import Presentation, Tag
+from presentation.permissions import IsAdminOrOwner, IsAdminOrReadOnly
+from presentation.serializers import (
+    InputPresentationSerializer,
+    OutputPresentationSerializer,
+    TagSerializer,
+)
+from presentation.services import PresentationService
+
+
+class PresentationViewSet(ModelViewSet):
+    queryset = Presentation.objects.all()
+    serializer_class = OutputPresentationSerializer
+    permission_classes = [IsAdminOrOwner]
+    services = PresentationService()
+
+    def create(self, request, *args, **kwargs):
+        serializer = InputPresentationSerializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+        instance = self.services.create_presentation(
```

Comment:
`instance` isn't so descriptive, try to aim for better variable names
```suggestion
        presentation = self.services.create_presentation(
```